site/docs/patterns/international-phone-number-input.mdx

```diff
@@ -0,0 +1,124 @@
+---
+title: International Phone Number Input
+layout: DetailPattern
+sidebar:
+  label: International Phone Number Input
+aliases:
+  - /salt/patterns/international-phone-number-input
+data:
+  resources:
+    [
+      {
+        href: "https://www.figma.com/file/HkXrrf1rbYATx6ggGG4Pov/(WIP)-International-Phone-Number?node-id=1356%3A3430&mode=design",
+        label: "Templates",
+        internal: true,
+      },
+    ]
+---
+
+International Phone Number Input is a pattern that gives users a set of form fields to input phone number information for specific countries.
+
+## Pattern scope
+
+The pattern is a form that displays a relevant phone number field for a user-selected location. You can use it to provide consistent design when implementing International Phone Number Input interactions across your applications.
+
+We provide design guidance on how you can use Salt form control components to achieve the pattern. Your team can take further steps that fall outside its scope. For example, check with your tech partners if it’s possible to auto-detect user location, using data that can support auto-generating number placeholders, and save the entered number when the user changes the country.
```

Comment:
"check with your tech partners if it’s possible to auto-detect user location, using data that can support auto-generating number placeholders, and save the entered number when the user changes the country."

I'm not really sure what this means. Are all these actions inclusive or should this say "or" rather than "and" @yunjungyeh

Reply:
Hey Josh, they should be considered different enhancements. @mikearildbrown @sion-docs would you say using "or" in this case is more precise?

Reply:
Text updated using 'or' and revised punctuation.